Arelle/Arelle
# Publish Type Hints in Python Package

#### Reason for change
See https://github.com/Arelle/ixbrl-viewer/pull/678#issuecomment-2182905387

#### Description of change
This marker file identifies the Arelle Python package as a typed library so users of the Python package can validate their use of Arelle with the type hints in the repo.

#### Steps to Test
* Checkout Paul's ixbrl-viewer branch:
	* `git checkout -b paulwarren-wk-incremental-builder master`
	* `git pull https://github.com/paulwarren-wk/ixbrl-viewer.git incremental-builder`
* Install dev dependencies:
	* `pip install .[arelle,dev]`
* Verify mypy throws an error:
	* `mypy iXBRLViewerPlugin --namespace-packages`
* Install Arelle from this branch:
	* `pip install git+https://github.com/austinmatherne-wk/Arelle.git@type-package`
* Rerun mypy and confirm that the error is no longer raised:
	* `mypy iXBRLViewerPlugin --namespace-packages`

**review**:
@Arelle/arelle
